Gemfile

```diff
@@ -32,6 +32,8 @@ gem 'rubocop', '>= 1.0', '< 2.0'
 
 gem 'devise'
 
+gem 'cancancan'
+
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: %i[mri mingw x64_mingw]
```

Comment:
- [ ] The _rswag-api_ and the _rswag-ui_ gems are missing in your project. These _gems_ are required for this project. Kindly ensure to add those gems to your project for testing purposes.

Reply:
Hi @LOctavio 
Kindly note that this requirement (rswag-api and the rswag-ui gems) is actually a requirement for another milestone. Sorry I missed this step during my review